src/atom.coffee

```diff
@@ -207,12 +207,6 @@ class Atom extends Model
   #
   # Call after this instance has been assigned to the `atom` global.
   initialize: ->
-    # Disable deprecations unless in dev mode or spec mode so that regular
-    # editor performance isn't impacted by generating stack traces for
-    # deprecated calls.
-    unless @inDevMode() or @inSpecMode()
-      require('grim').deprecate = ->
```

Comment:
@nathansobo I'm guessing this probably should have been removed after https://github.com/atom/grim/pull/4 was merged and just never was. Any reason not to :fire: it now?